### PR TITLE
SDK-1110 task manifest phase 2

### PR DESCRIPTION
**Pull request by sergeyspryker**

### CODE REVIEW

File: src/Enum/ValueTypeEnum.php [hunk 1]
```diff
@@ -38,4 +38,23 @@ final class ValueTypeEnum
...
     public const TYPE_UUID = 'uuid';
+
+    /**
+     * {@inheritDoc}
+     *
+     * @api
+     *
+     * @return array<string>
+     */
+    public static function getAllValueTypes(): array
```
Comment (vol4onok) [inline src/Enum/ValueTypeEnum.php:49]: we don't need it in contracts and group lists for settings and placeholders. They have different types

**Review by vol4onok — APPROVED** [review]

**Review by vol4onok — APPROVED** [review]

**Review by vol4onok — CHANGES_REQUESTED** [review]
see comment

File: src/Enum/ValueTypeEnum.php (hunk 1 above)
Comment (vol4onok) [reply src/Enum/ValueTypeEnum.php:49]: Or split this class static::TYPE_PATH,  static::TYPE_UUID, should be SettingValueTypeEnum extends ValueTypeEnum

Comment (vol4onok) [reply src/Enum/ValueTypeEnum.php:49]: @pavelmaksimov25 We can do it now or in the next epic

**Review by vol4onok — APPROVED** [review]

**Comment by sergeyspryker** [comment]
After discussions decided to implement it in separate epic

**Pull request closed without merging**